frontend/apps/reader/modules/readergesture.lua

```diff
@@ -726,6 +726,11 @@ function ReaderGesture:gestureAction(action, ges)
             -- NB Normal widgets should use NetworkMgr:promptWifiOn()
             -- This is specifically the toggle wifi action, so consent is implied.
             NetworkMgr:turnOnWifi()
+
+            UIManager:show(InfoMessage:new{
+                text = _("Wifi enabled."),
```

Comment:
Btw, this message would be somewhat presumptuous imo, but this specific order could also have odd results on some platforms. Here's a video of what currently happens on Kobo.

[2019-03-13 23.28.39.mp4.zip](https://github.com/koreader/koreader/files/2963914/2019-03-13.23.28.39.mp4.zip)

(Just to clarify why I wrote it the way I did.)